server/test/user.route.v1.test.ts

```diff
@@ -18,56 +18,107 @@ describe("User Route v1", () => {
     let user1: SocoboUser = new SocoboUser();
```

Comment:
I think the Tests are OK. I would just add some unit tests. I can do this anytime later, maybe in another PR   in Order to get this one done. Do we need an issue to collect pending tests?